lighthouse/constants.py

```diff
@@ -10,9 +10,14 @@
 FIELD_PLATE_BARCODE = "plate_barcode"
 FIELD_COG_BARCODE = "cog_barcode"
 FIELD_DATE_TESTED = "Date Tested"
+FIELD_CQ_1 = "ch1_cq"
```

Comment:
I named these after the fields in crawler, so FIELD_CH1_CQ, FIELD_CH1_RESULT, FIELD_CH1_TARGET etc., dunno if it's worth keeping things consistent?
Also there are 4 channels, assume 4 is not relevant here?

Reply:
Yep OK will change. Don't think we need 4 here, as not filtering on it